arangod/MMFiles/MMFilesCollection.cpp

```diff
@@ -543,6 +544,7 @@ MMFilesCollection::MMFilesCollection(LogicalCollection* logical,
     _indexes.emplace_back(idx);
   }
 */
+  TRI_ASSERT(!ServerState::instance()->isCoordinator());
```

Comment:
can we remove the comment above?

Reply:
I don't now, right now there seems to be no difference in the implementation.